### DISCUSSION TITLE
Manifold integration & display

### DISCUSSION THREAD

**Discussion by Bentoon** [f8n/fnd-protocol#128] [Q&A] [answered]
I'm so excited by this and have been watching for a while...
As you roll this out, please allow the loading of IPFS/ARWEAVE...

My manifold contracts are dynamic nfts so I'm really hoping the foundation protocol can display them as such, 
https://opensea.io/assets/ethereum/0xfdca97a72a628529dcafcd876611246a3fed14cb/5
Some marketplaces get this right and others don't. 
But it will make a big difference @HardlyDifficult Thanks for your help

Jb
https://foundation.app/@JohnBenton

**Comment by HardlyDifficult** [comment, marked as answer]
@corif8n could you take a look?

**Reply by corif8n** [reply]
Hi @Bentoon! Looking at the assets, I don't believe we'll be able to support them in their full dynamic state (for now!). We're actively working on expanding our coverage, but no ETA on when text/html assets will be supported.

**Reply by Bentoon** [reply]
@corif8n Thanks for looking into it. 
Did you try to view the work on Safari (WebGL won't work yet) or is it a protocol w foundation.app?
Zora & even Gallery displays but Rarible & Coinbase only list
https://gallery.so/Benton/28hiJsbsRzV3LUvMT9hSgxhwOSe/29lC5V0bs3lnRonVBHOcsdH4OSZ

I ask because I'm writing an article for SuperRare on Dynamic NFT's and see html / webGL are gaining a lot of momentum as manifold/ generative/pfps grow.
But thanks for your reply, I appreciate your help and hope this unveils soon
~be

**Reply by corif8n** [reply]
@Bentoon looking forward to checking out the article!
We aim to provide the best UX within foundation.app and supporting dynamic NFTs in a performant way is top of mind. This is where the constraint currently is.
We're working in that direction though, don't worry!

**Reply by Bentoon** [reply]
Thanks, Cori
I appreciate it. You all are doing an amazing job 
Looking forward,
~be
